scripts/add-product-to-blob.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+from os import path
+import sys
+
+import simplejson as json
+import requests
+from requests.exceptions import RequestException
+
+if __name__ == "__main__":
+    from optparse import OptionParser
+    doc = "%s --u uri -n username -p password" % sys.argv[0]
+    parser = OptionParser(doc)
+    parser.add_option("-u", "--url", dest="url", type="string", default=False,
+                      action="store", help="URL to the running admin API")
+    parser.add_option("-n", "--username", dest="username", type="string", default=False,
+                      action="store", help="Username for the Admin interface")
+    parser.add_option("-p", "--password", dest="password", type="string", default=False,
+                      action="store", help="Password for the Admin interface")
+    options, args = parser.parse_args()
+
+    if not options.url or not options.username:
+        print "url and auth is required"
+        sys.exit(1)
+
+    url = 'http://' + options.url
+    data = requests.get(url + '/api/releases')
+    releases = data.json()['releases']
+
+    unsuccessful_releases = []
+    for release in releases:
+        try:
+            client = requests.Session()
+            data = client.get(url + '/api/releases/' + release['name'])
+            blob = data.json()
+            csrf_token = data.headers['X-CSRF-Token']
+            if 'product' not in blob.keys():
```

Comment:
Having this condition was a good idea, because it makes this safe and easy to rerun. Nice job!